src/hooks/usePagination.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { client } from 'client';
 import appConfig from 'app.config';
 
-const uniqBy = (arr, pred) => {
+export const uniqBy = (arr, pred) => {
```

Comment:
Given now that we export this, it makes sense to create a `utils` folder and move this function there.